workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/schema/zod.ts

```diff
@@ -113,6 +113,11 @@ const zodSchema = {
 			tags: z.array(z.any()).nonempty(),
 		}),
 		storefront: z.object({images: z.array(z.any()).min(1).max(10)}),
+		support: z.object({
+			email: z.string().email(),
+			phone: z.string().min(8),
+			publisherWebsiteURL: z.string().url(),
+		}),
```

Comment:
@CalebJamesH is good to have this options here, but as you did this 3 fields validations, could you also add to the others?

For now, make all of them optional

using .optional()